api/app/signals/settings/base.py

```diff
@@ -124,11 +124,16 @@
 ]
 
 # Database settings
-DATABASE_NAME = os.getenv('DATABASE_NAME', 'signals')
-DATABASE_USER = os.getenv('DATABASE_USER', 'signals')
-DATABASE_PASSWORD = os.getenv('DATABASE_PASSWORD', 'insecure')
-DATABASE_HOST = os.getenv('DATABASE_HOST_OVERRIDE')
-DATABASE_PORT = os.getenv('DATABASE_PORT_OVERRIDE')
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.contrib.gis.db.backends.postgis',
+        'NAME': os.getenv('DATABASE_NAME', 'signals'),
+        'USER': os.getenv('DATABASE_USER', 'signals'),
+        'PASSWORD': os.getenv('DATABASE_PASSWORD', 'insecure'),
+        'HOST': os.getenv('DATABASE_HOST_OVERRIDE'),
+        'PORT': os.getenv('DATABASE_PORT_OVERRIDE')
```

Comment:
Maybe this is also a good opportunity to rename `DATABASE_HOST_OVERRIDE` to `DATABASE_HOST` and `DATABASE_PORT_OVERRIDE` to `DATABASE_PORT`?